spec/p2p/messages/state-sync.md

```diff
@@ -70,16 +71,41 @@ possibly from a different peer.
 
 The ABCI application is able to request peer bans and chunk refetching as part of the ABCI protocol.
 
+### LightBlockRequest
```

Comment:
There is a proto directory in this repo as well. Could you add these there?

Reply:
Yup. I didn't see that 👍